### PR TITLE
document normalizers in global-phase

### PR DESCRIPTION
**Pull request by arnej27959**
* also mention query property ranking.globalPhase.rerankCount
* and how to force sub-expressions to be computed on content nodes

@kkraune please review
@jobergum please review
@geirst @havardpe FYI

### CODE REVIEW

File: en/phased-ranking.html [hunk 1]
```diff
@@ -181,6 +186,70 @@ <h2 id="global-phase">Using a global-phase expression</h2>
 }
 </pre>
 
+<h2 id="normalizers">Cross-hit normalization in global-phase</h2>
```
Comment (kkraune) [inline en/phased-ranking.html:189]: Id should be same as heading - cross-hit-etc

**Review by kkraune — APPROVED** [review]
Great!

File: en/phased-ranking.html [hunk 2]
```diff
@@ -181,6 +186,70 @@ <h2 id="global-phase">Using a global-phase expression</h2>
...
+  This means that first, the input (<em>my_function_or_feature</em>)
+  is computed or extracted from each hit that global-phase will
+  rerank; then the normalization step is applied; afterwards when
+  computing the actual global-phase expression the normalized output
+  is used.
+  As an example, assume some text fields with bm25 enabled, an onnx
+  model like above, and a "popularity" numeric attribute:
+</p>
+<pre>
+    rank-profile with-normalizers inherits with-global-model {
```
Comment (jobergum) [inline en/phased-ranking.html:213]: Were does the `with-global-model` come from? Is this something that we have built in now to signal global phase?

**Review by jobergum — APPROVED** [review]

**Comment by jobergum** [comment]
Great! Please see my comment

**Review by kkraune — APPROVED** [review]

**Pull request merged**